rle-bitpacked.md

```diff
@@ -0,0 +1,120 @@
+# RLE-Bitpacked hybrid encoder
+
+The RLE-Bitpacked hybrid encoder is a parquet-specific encoder that combines
```

Comment:
"encoding" rather than "encoder". The encoding is implemented by an encoder + decoder.